examples/website-demos/social-messaging/src/components/ChannelPreview/utils.tsx

```diff
@@ -0,0 +1,77 @@
+import type { Channel, ChannelMemberResponse } from 'stream-chat';
+
+import { Avatar } from 'stream-chat-react';
+
+export const AvatarGroup = ({ members }: { members: ChannelMemberResponse[] }) => {
+  if (members.length === 1) {
+    return <Avatar image={members[0].user?.image as string || ''} size={56} />;
```

Comment:
i recently went back and added to the old social app, but you should put a name field on here so if there's no image we at least display the initial on the blue background